README.v0.md

```
@@ -0,0 +1,247 @@
+
+The goal of BAML is to give you the expressiveness of English, but the structure of code.
+
+Full [blog post](https://www.boundaryml.com/blog/ai-agents-need-new-syntax) by us.
+
+
+## Conclusion
+
+As models get better, we'll continue expecting even more out of them. But what will never change is that we'll want a way to write maintainable code that uses those models. The current way we all just assemble strings is very reminiscent of the early days PHP/HTML soup in web development. We hope some of the ideas we shared today can make a tiny dent in helping us all shape the way we all code tomorrow.
+
+## FAQ
+|   |   |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)